Std/Data/HashMap/Lemmas.lean

```diff
@@ -4,12 +4,15 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Scott Morrison
 -/
 import Std.Data.HashMap.Basic
+import Std.Data.Array.Lemmas
 import Std.Util.ProofWanted
 
 namespace Std.HashMap
 
-@[simp] proof_wanted empty_find? [BEq α] [Hashable α] {a : α} :
-    (∅ : HashMap α β).find? a = none
+@[simp] theorem empty_buckets [BEq α] [Hashable α] : (∅ : HashMap α β).1.buckets.1 = mkArray 8 AssocList.nil := rfl
```

Comment:
This should be in the `HashMap.Imp` namespace as it is talking about implementation details.

Reply:
Fixed + split in two.